### PR TITLE
[Frontend] UI/pool interface

### PR DESCRIPTION
**Pull request by novelja**
## Pool and Add Liquidity UI styles

**Pool List (`/pool`)**
- [x] need a way to read the list of pools user joined

**Add Liquidity**
- [x] confirmation screens

### CODE REVIEW

File: ui/app/src/views/PoolPage.vue [hunk 1]
```diff
@@ -2,34 +2,68 @@
...
-  color: #333;
-  background: #aaa;
-  margin-bottom: 1rem;
+<style scoped lang="scss">
+.heading {
+  display: flex;
+  align-items: center;
+
+  h3 {
+    @include title16;
```
Comment (ryardley) [inline ui/app/src/views/PoolPage.vue:56]: Have we got an explanation of these different titles etc. anywhere? Would love to see a styleguide page or our components displayed in storybook.

File: ui/app/vue.config.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-          @import "normalize-scss";
-          @import "@/scss/reset.scss";
-          @import "@/scss/typography.scss";
-          @import "@/scss/variables.scss";
-          @import "@/scss/utilities.scss";
-          @import "@/scss/mixins.scss";
-        `
-      }
+    css: {
+        extract: {
```
Comment (ryardley) [inline ui/app/vue.config.js:10]: Is this reformatting required? Maybe I am wrong but it looks like prettier isn't setup correctly or wasn't setup correctly either on the project or in editor.

File: ui/core/src/hooks/addLiquidityCalculator.ts [hunk 3]
```diff
@@ -105,11 +105,11 @@ export function usePoolCalculator(input: {
     const bAmount = toField.fieldAmount.value;
     if (!aAmount || !bAmount) return "";
     if (bAmount.equalTo("0")) return "";
-    return `${aAmount
+    return `<span class="number">${aAmount
```
Comment (ryardley) [inline ui/core/src/hooks/addLiquidityCalculator.ts:108]: I don't think this markup should be here. We would be much better off returning primitives from here and formatting them in the consumer.

**Review by ryardley — CHANGES_REQUESTED** [review]
Nice work. Main concern I have is adding formatting to the liquidity calculator as that really should have no knowledge of how it's output is used.

**Comment by ryardley** [comment]
> need a way to read the list of pools user joined

There is an api that has been built for this. I can hook it up for you. cc @59023g in the meantime it might work to just mock it using the equivalent output of `api.MarketService.getPools()`

File: ui/app/src/components/shared/PriceCalculation.vue [hunk 4]
```diff
@@ -6,15 +6,16 @@
...
 <script>
 export default {
-
 }
 </script>
 
 <style lang="scss">
 .price-calculation {
-  min-height: 120px;
+  height: 120px;
```
[the remark this answers is not in the document]
Comment (novelja) [reply ui/app/src/components/shared/PriceCalculation.vue:14]: Yeah, I also like to keep these dynamic. But from my understanding, Joerg aims to keep the height of the whole panel at a fixed height, which further implies we should make some of these components at a fixed height also. He also uses scrollbars extensively throughout the design for cases such as this.

File: ui/app/src/views/PoolPage.vue (hunk 1 above)
Comment (novelja) [reply ui/app/src/views/PoolPage.vue:56]: Not yet. There are a lot of different copy/type styles and with the new designs, new styles are emerging. I'll look into the best solution to organize these.

File: ui/core/src/hooks/addLiquidityCalculator.ts (hunk 3 above)
Comment (novelja) [reply ui/core/src/hooks/addLiquidityCalculator.ts:108]: Agree. I'll need this as two separate strings for styling purposes. Is it fine if I returned an object here with `value` and `copy`?

File: ui/app/src/components/shared/PriceCalculation.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (ryardley) [reply ui/app/src/components/shared/PriceCalculation.vue:14]: We are going to have to make this responsive though anyway. I think we should be careful here.

File: ui/core/src/hooks/addLiquidityCalculator.ts (hunk 3 above)
Comment (ryardley) [reply ui/core/src/hooks/addLiquidityCalculator.ts:108]: yup it should return whatever is the easiest for the UI without knowing anything about the way the UI is rendering the info

**Comment by ryardley** [comment]
Ok @novelja if you can rebase develop with this you should have access to read the list of pools user joined by using the following:

```ts
await actions.clp.getLiquidityProviderPools()
```

**Review by 59023g — APPROVED** [review]

**Comment by 59023g** [comment]
Tickets created for the following: 
- Display Pool Tokens Received on Confirmation Screen https://app.asana.com/0/1198643968827245/1199660948423348
- Display Liquidity Provider's Pool Share https://app.asana.com/0/1198643968827245/1199660948423343

**Review by ryardley — APPROVED** [review]

**Pull request merged**